_config.yml

```diff
@@ -21,6 +21,9 @@ url: "https://learn.circuitverse.org" # the base hostname & protocol for your si
 permalink:
 exclude: ["node_modules/", "*.gemspec", "*.gem", "Gemfile", "Gemfile.lock", "package.json", "package-lock.json",  "script/", "LICENSE.txt", "lib/", "bin/", "README.md", "Rakefile", "vendor"]
 
+
+
+
```

Comment:
✂️

Reply:
Sorry, I was working on my phone. It should be fixed now